scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
> I think that real tests with our image and bay-create are giving us better accurate tests for users

@mjura What do you mean with "real tests" ? What's not real with tempest tests? Tempest is already creating a baymodel and a bay. It's just increasing the time the CI needs to finish. And only benefit I see is that the magnumclient is tested. But a simple "magnum list" would imo be enough here.